angular/projects/spark-angular/src/lib/components/sprk-tabs/sprk-tabs.stories.ts

```diff
@@ -45,15 +39,69 @@ only one content panel will be visible at a time.
 };
 
 const modules = {
+  imports: [SprkTabsModule, SprkTabsPanelModule, SprkTabsButtonModule],
+};
+
+export const defaultStory = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-tabs idString="tabs-1">
+      <button
+        sprkTabsButton
+        analyticsString="Tab: 1"
+        data-id="tab-1"
+      >
+        Tab 1
+      </button>
+      <button
+        sprkTabsButton
+        data-id="tab-2"
+        [isDefaultActive]=true
+      >
+        Tab 2
+      </button>
+      <button
+        sprkTabsButton
+        data-id="tab-3"
+      >
+        Tab 3
+      </button>
+      <div sprkTabsPanel>
+        <p>Tab 1 Content Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+      </div>
+      <div [isDefaultActive]=true sprkTabsPanel>
+        <p>Tab 2 Content Sed quis rhoncus ipsum. Nulla euismod nisi est, vel
+        consequat ante consectetur in.</p>
+      </div>
+      <div sprkTabsPanel>
+        <p>Tab 3 Content Ut interdum dictum est at ornare. Nam nec dapibus nibh.
+        Integer venenatis ex eu mi euismod, non ultricies lacus venenatis.</p>
+      </div>
+    </sprk-tabs>
+  `,
+});
+
+defaultStory.story = {
+  name: 'Default',
+  parameters: {
+    jest: [
+      'sprk-tabs.component',
+      'sprk-tabs-panel.directive',
+      'sprk-tabs-button.directive',
+    ],
+  },
+};
+
+const modules_deprecated = {
   imports: [
     SprkTabbedNavigationModule,
     SprkTabbedNavigationPanelModule,
     SprkTabbedNavigationTabModule,
   ],
 };
 
-export const defaultStory = () => ({
-  moduleMetadata: modules,
+export const deprecated = () => ({
```

Comment:
We usually called these "Legacy (Deprecated)"
![image](https://user-images.githubusercontent.com/4342363/104760681-b4015e00-572f-11eb-9423-2981a4840fda.png)

Reply:
✅